### PR TITLE
Team confirm modal style tweaks

### PR DESCRIPTION
**Pull request by buoyad**
I'm testing with a team with a super long name so a lot of this is in the interest of that.

- Various fixes around long teamnames in these modals
- Fix issues with header icons
- Use correct text type for header prompts (and don't lineclamp them, they are UI text)

- [x] storyshots before merge

 cc @keybase/y2ksquad

### CODE REVIEW

File: shared/teams/confirm-modals/confirm-kick-out.tsx [hunk 1]
```diff
@@ -84,38 +84,45 @@ const ConfirmKickOut = (props: Props) => {
...
-        sizeType="Small"
-      />
+      <Kb.Box2 direction="horizontal" centerChildren={true} style={styles.iconContainer}>
+        <Kb.Icon
+          type="iconfont-block"
+          color={Styles.globalColors.white}
+          fontSize={14}
+          style={styles.headerIcon}
+        />
+        {/* boxStyle={members.length <= 5 ? styles.iconContainerSingle : styles.iconContainer} */}
```
Comment (jakob223) [inline shared/teams/confirm-modals/confirm-kick-out.tsx:102]: 🔥

File: shared/teams/confirm-modals/confirm-kick-out.tsx [hunk 2]
```diff
@@ -84,38 +84,45 @@ const ConfirmKickOut = (props: Props) => {
...
   const header = (
     <Kb.Box style={styles.positionRelative}>
       <Kb.AvatarLine usernames={members} size={64} layout="horizontal" maxShown={5} />
-      <Kb.Icon
-        boxStyle={members.length <= 5 ? styles.iconContainerSingle : styles.iconContainer}
-        type="iconfont-block"
-        style={styles.headerIcon}
-        sizeType="Small"
-      />
+      <Kb.Box2 direction="horizontal" centerChildren={true} style={styles.iconContainer}>
```
Comment (jakob223) [inline shared/teams/confirm-modals/confirm-kick-out.tsx:95]: This is finnicky - did you test with 1,2, and many?

File: shared/teams/confirm-modals/delete-channel/index.tsx [hunk 3]
```diff
@@ -13,8 +14,8 @@ type Props = Container.RouteProps<{
 
 const Header = () => (
   <>
-    <Kb.Icon type={'icon-teams-channel-64'} />
-    <Kb.Icon type="icon-team-delete-28" style={{marginRight: -60, marginTop: -20, zIndex: 1}} />
```
Comment (jakob223) [inline shared/teams/confirm-modals/delete-channel/index.tsx:17]: 😱

File: shared/teams/confirm-modals/confirm-kick-out.tsx (hunk 2 above)
Comment (buoyad) [reply shared/teams/confirm-modals/confirm-kick-out.tsx:95]: Not many enough actually. Fixed.

**Pull request merged**